extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -215,6 +215,31 @@ describe('integration test: a4a', () => {
         });
       });
 
+  it('should collapse slot when creative response has code 204', () => {
+    headers = {};
+    headers[SIGNATURE_HEADER] = validCSSAmp.signature;
+    mockResponse = {
+      arrayBuffer: () => utf8Encode(validCSSAmp.reserialized),
+      bodyUsed: false,
+      headers: new FetchResponseHeaders({
+        getResponseHeader(name) {
+          return headers[name];
+        },
+      }),
+      status: 204,
+    };
+    xhrMock.withArgs(TEST_URL, {
+      mode: 'cors',
+      method: 'GET',
+      credentials: 'include',
+    }).onFirstCall().returns(Promise.resolve(mockResponse));
+    const forceCollapseStub =
+        sandbox.stub(MockA4AImpl.prototype, 'forceCollapse');
+    return fixture.addElement(a4aElement).then(unusedElement => {
+      expect(forceCollapseStub).to.be.calledOnce;
```

Comment:
I inspected the DOM manually after collapse, but couldn't find any condition that would suggest that a collapse has taken place (unless I'm missing something). 

How can I check that an internal variable's function was called? As far as I've tried, I couldn't figure out a way to create a stub on a private, imported, object (the uihandler).